src/panels/lovelace/hui-root.ts

```diff
@@ -228,7 +228,18 @@ class HUIRoot extends LitElement {
                   >
                     ${this.lovelace!.config.views.map(
                       (view) => html`
-                        <paper-tab aria-label="${view.title}">
+                        <paper-tab
+                          aria-label="${view.title}"
+                          class="${classMap({
+                            "hide-tab": Boolean(
+                              !this._editMode &&
+                                view.hide &&
+                                (view.hide === true ||
+                                  (Array.isArray(view.hide) &&
+                                    view.hide.includes(this.hass!.user!.name)))
```

Comment:
This value can be changed in the UI. Should be user id.

Reply:
How do I grab that?